data/json/martialarts.json

```
@@ -157,7 +157,14 @@
         "name": "Boxing Stance",
         "description": "A solid stance allows you block more damage than normal and deliver better punches.\n\n+2 Bash damage, Blocked damge reduced by 50% of Strength.",
         "unarmed_allowed": true,
-        "flat_bonuses": [ { "stat": "block", "scaling-stat": "str", "scale": 0.5 }, { "stat": "damage", "type": "bash", "scale": 2.0 } ]
+        "flat_bonuses": [
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "bash", "scale": 2.0 },
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "cut", "scale": 2.0 },
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "stab", "scale": 2.0 }
```

```suggestion
          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
          { "stat": "damage", "type": "bash", "scale": 2.0 },
          { "stat": "damage", "type": "cut", "scale": 2.0 },
          { "stat": "damage", "type": "stab", "scale": 2.0 }
```
Definitely just a copypaste error it looks like.